packages/flutter_tools/test/commands.shard/hermetic/http_host_availability_validator_test.dart

```diff
@@ -0,0 +1,96 @@
+// ignore: import_of_legacy_library_into_null_safe
```

Comment:
Don't ignore this error, opt the doctor validator out of null safety like the other tests